### DISCUSSION TITLE
How to create internal server (Win10 - 64 bit) - use Audiogridder on the same machine ???

### DISCUSSION THREAD

**Discussion by John5897** [apohl79/audiogridder#752] [Help]
Hello to you, I am asking for help to use "Audiogridder" on one and the same machine and not with another station as a slave.
My idea is to use several plugins apart from my DAW (Reaper) because some use a lot of processor resources and are poorly distributed in multiprocessing in Reaper. So I am almost certain that it is possible to create a virtual internal network on the same machine and to run the audiogridder on the same station, so as to use my plugs in outside the DAW, because "Audiogridder" manages them much better. VST3 plugins by distributing them better in processor computing. Could someone help me to create a network under Win 10 - 64 bit ??? Here, thank you in advance to this one who will want to do it. Goodbye. - Jerome

**Comment by apohl79** [comment]
You don't have to create any local network to use AG on the same box. Just install server and plugin on the same machine and run the server. You might want to enable the "local mode" in the server. In your DAW load AG plugins and select the server, as they should find it.
